msteams-platform/sbs-botbuilder-linkunfurling.yml

```diff
@@ -83,7 +83,8 @@ items:
        :::image type="content" source="~/assets/images/Teams-link-unfurling/review-plus-create.png" alt-text="Select resource Azure bot":::
        
        You can also create a new resource group (select **Create new** > enter resource name > select **OK**).
-    
+       For more information see [Create resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#create-resource-groups)
```

Comment:
Remove the steps from the parenthesis. Use the correct link (Link to Docs page by URL) tag in VSC.

Reply:
The link is removed as suggested.